packages/vuetify/src/components/VDataTable/VDataTableHeaders.tsx

```diff
@@ -203,7 +222,78 @@ export const VDataTableHeaders = genericComponent<VDataTableHeadersSlots>()({
       )
     }
 
+    const VDataTableMobileHeaderCell = () => {
+      const headerProps = mergeProps(props.headerProps ?? {} ?? {})
+
+      const displayItems = computed<ItemProps['items']>(() => {
+        return columns.value.filter(column => column?.sortable)
+      })
+
+      return (
+        <VDataTableColumn
+          tag="th"
+          class={[
+            ...headerCellClasses.value,
+            loaderClasses.value,
+          ]}
+          colspan={ headers.value.length + 1 }
+          { ...headerProps }
+        >
+          <div class="v-data-table-header__content">
+            <VSelect
+            chips
+            class={[
+              'my-2',
+              'v-data-table__mobile-sort-select',
+            ]}
+            clearable
+            color="primary"
```

Comment:
No default colors

Reply:
Should we add a prop so a user can change the color, or just remove it?

Reply:
Just remove for now.